### PR TITLE
Yolo/improve installation poetry

### PR DESCRIPTION
**Pull request by FirelightFlagboy**

### CODE REVIEW

File: .github/workflows/linux-ci.yml [hunk 1]
```diff
@@ -59,22 +59,10 @@ jobs:
...
-        run: |
-          set -eux
-          export POETRY_HOME="$HOME/.poetry"
-          mkdir -p "$POETRY_HOME"
-          curl -sSL https://install.python-poetry.org | python3 - --version=${{ env.poetry-version }}
-          echo "$POETRY_HOME/bin" >> "$GITHUB_PATH"
-
-      - uses: actions/setup-python@c4e89fac7e8767b327bbad6cb4d859eda999cf08  # pin v4.1.0
-        id: setup-python
+      - uses: ./.github/actions/setup-poetry
```
Comment (touilleMan) [inline .github/workflows/linux-ci.yml:62]: Renaming the action `setup-python-poetry` would be more explicit (and it's already the name in action.yml)

**Review by touilleMan — CHANGES_REQUESTED** [review]
This is much, much more readable ! \o/

File: .github/workflows/linux-ci.yml (hunk 1 above)
Comment (FirelightFlagboy) [reply .github/workflows/linux-ci.yml]: done

**Review by touilleMan — APPROVED** [review]
:ok_hand:

**Pull request merged**